test/xspec.bats

```diff
@@ -173,5 +173,5 @@
     run ../bin/xspec.sh ../test/xspec-46.xspec
```

Comment:
This test depends on a new directory. So it would be helpful if there is a comment immediately before this test.
Like the comment [`"# this test must run first to create xspec directory"`](https://github.com/xspec/xspec/blob/ba5f97c903d0022d322d7af7e379b6fbcaf35ee6/test/xspec.bats#L73) before the test `"invoking code coverage with Saxon9EE creates test stylesheet"`.